frontend/web/components/ProjectFilter.tsx

```diff
@@ -30,7 +32,7 @@ const ProjectFilter: FC<ProjectFilterType> = ({
         (data || [])?.map((v) => ({ label: v.name, value: `${v.id}` })),
       )}
       onChange={(value: { value: string; label: string }) =>
-        onChange(value?.value || '')
+        onChange(projectComplete ? value : value?.value ? value.value : '')
```

Comment:
Unclear what this is doing, why is this adjusting the project filter

Reply:
This accessory is evaluated solely when utilizing the project filter on the GitHub configuration page to return the correct value. Therefore, renaming it will enhance clarity.

Reply:
Adjusting a reusable component just for the sake of a GitHub integration doesn't seem to make sense. Can't you just pass value appropriately ?